src/beatmap.c

```diff
@@ -185,6 +185,28 @@ static void parse_one_hit(char *line, struct oshu_hit **hit)
 }
 
 /**
+ * Parse #oshu_hit::combo and #oshu_hit::combo_seq of a single #oshu_hit
+ */
+static void parse_hit_combo(struct oshu_hit *previous, struct oshu_hit *hit)
```

Comment:
Feels weird to call this function *parse* even though it doesn't receive any kind of text. Maybe call it `compute_combo` or `compute_hit_combo`?

And you forgot a dot in the documentation 🌵